{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}/.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,87 @@
+{% raw %}name: Publish python distribution to PyPI
+on: push
```

Comment:
Do you want to use less frequent action?
1. Like this on_dispatch https://github.com/microsoft/python-package-template/blob/main/.github/workflows/publish.yml
2. Like this on push tag https://github.com/rochacbruno/python-project-template/blob/main/.github/workflows/release.yml
3. Like my case - after publication the release. It's useful with "release-drafter". https://github.com/Nov1kov/sbermarket-api/tree/main/.github/workflows

Reply:
Oh, I see your "if" for workflow. Anyway please keep in mind my comment above.